### PR TITLE
[WEB-288] update access control guide

### PR DESCRIPTION
**Pull request by pizzasaurusrex**
Name of feature:
Group Sync and Group Principal to Certificate Documentation

Why is this important to the project (if not answered above):
We will be releasing group sync web ui and just released a new version of step-ssh that supports it (yay!). But we should let people know more about it somewhere. THIS IS THE PLACE!!

Is there documentation on how to use this feature? If so, where?
Right friggen here my friends.

In what environments or workflows is this feature supported?
hosts running step-ssh version 0.22.0 or greater

In what environments or workflows is this feature explicitly NOT supported (if any)?
step-ssh version step-ssh version 0.21.0 and below.

💔Thank you!

### CODE REVIEW

File: src/pages/docs/ssh/acls.mdx [hunk 1]
```diff
@@ -46,18 +46,28 @@ Sign in at `https://smallstep.com/app/[Team ID]`
...
+* Choose the "GROUPS" tab
 
 ![](/graphics/quickstart/acl-select-idp-group.png)
 
 * Select the User Group that you want to configure for access control. You will see the user group detail page with a list of existing Host Grants and a form to add additional grants.
+* Select "Edit" from the kebab menu to the right of the Group's name.
   * Enter the tag values from Step 1
   * If Sudo access is permitted for this user group, select the "Allow Sudo" checkbox
   * Click the checkbox to enable access control
+* **Optional**. Assign Posix Groups. Smallstep now supports posix group support on hosts running step-ssh version 0.22.0 and greater so SSH Pro users can have more granular access to Hosts.
```
Comment (tashian) [inline src/pages/docs/ssh/acls.mdx]:
```suggestion
* **Optional**. Assign POSIX Groups. Smallstep now supports POSIX groups on hosts running `step-ssh` version 0.22.0 or greater. SSH Professional users can have more granular access to Hosts. 
```

I'm not 100% clear on what the last sentence about "granular access to hosts" means

File: src/pages/docs/ssh/acls.mdx [hunk 2]
```diff
@@ -46,18 +46,28 @@ Sign in at `https://smallstep.com/app/[Team ID]`
...
 
 ![](/graphics/quickstart/acl-select-idp-group.png)
 
 * Select the User Group that you want to configure for access control. You will see the user group detail page with a list of existing Host Grants and a form to add additional grants.
+* Select "Edit" from the kebab menu to the right of the Group's name.
   * Enter the tag values from Step 1
   * If Sudo access is permitted for this user group, select the "Allow Sudo" checkbox
   * Click the checkbox to enable access control
+* **Optional**. Assign Posix Groups. Smallstep now supports posix group support on hosts running step-ssh version 0.22.0 and greater so SSH Pro users can have more granular access to Hosts. 
+**Managed Groups** are posix groups that you want smallstep to create or delete on your Hosts.
```
Comment (tashian) [inline src/pages/docs/ssh/acls.mdx]:
```suggestion
**Managed Groups** are POSIX groups that you want Smallstep to create or delete on your Hosts.
```

File: src/pages/docs/ssh/acls.mdx [hunk 3]
```diff
@@ -46,18 +46,28 @@ Sign in at `https://smallstep.com/app/[Team ID]`
...
+**Unmanaged Groups** are posix groups such as  ***fax*** or  ***floppy*** that are default or already exist on the Host. Groups members will have permissions assigned to these groups.
```
Comment (tashian) [inline src/pages/docs/ssh/acls.mdx]:
```suggestion
**Unmanaged Groups** are the default POSIX groups such as  ***fax*** or  ***floppy*** that already exist on the Host. Groups members will have permissions assigned to these groups.
```

File: src/pages/docs/ssh/acls.mdx [hunk 4]
```diff
@@ -46,18 +46,28 @@ Sign in at `https://smallstep.com/app/[Team ID]`
...
+  * In the example below we are adding the group to one unmanaged group ***floppy*** and and managed group ***sre-team***
```
Comment (tashian) [inline src/pages/docs/ssh/acls.mdx]:
```suggestion
  * In the example below, we are adding users in the Smallstep group **okta-isvtest-ssh** to one unmanaged POSIX group (***floppy***) and one managed POSIX group (***sre-team***)
```

File: src/pages/docs/ssh/acls.mdx [hunk 5]
```diff
@@ -46,18 +46,28 @@ Sign in at `https://smallstep.com/app/[Team ID]`
...
+* **Optional**. Assign Certificate Principals. Smallstep now supports adding additional prinicals to user certificates. A **Principal** is a string that allows access to a host, or even a specific user on a host.
+Users in the assigned group will be able to access smallstep managed hosts as that user.
```
Comment (tashian) [inline src/pages/docs/ssh/acls.mdx]:
```suggestion
Users in the assigned group will be able to access Smallstep-managed hosts as that user, whether or not the user is managed by Smallstep. Adding a user as a certificate principal does not enable full lifecycle management for that user.
```

File: src/pages/docs/ssh/acls.mdx [hunk 6]
```diff
@@ -46,18 +46,28 @@ Sign in at `https://smallstep.com/app/[Team ID]`
...
+  * In the example below we are adding the principal ***craig*** to the Group.
```
Comment (tashian) [inline src/pages/docs/ssh/acls.mdx]:
```suggestion
  * In the example below we are adding the principal ***craig*** to certificates issued for the Group **okta-isvtest-ssh**.
```

File: src/pages/docs/ssh/acls.mdx [hunk 7]
```diff
@@ -46,18 +46,28 @@ Sign in at `https://smallstep.com/app/[Team ID]`
...
 
 ![](/graphics/quickstart/acl-grant-access.png)
 
 * All users in the group now have access to all hosts with the specified tag combination.
+
+
+**Posix Groups** Smallstep now gives SSH professional accounts the ability to map  Groups to Posix Groups on Hosts.
```
Comment (tashian) [inline src/pages/docs/ssh/acls.mdx]:
```suggestion
**POSIX Groups** SSH Professional accounts can map Smallstep Groups to POSIX Groups on Hosts. 
```

**Review by tashian — CHANGES_REQUESTED** [review]

File: src/pages/docs/ssh/acls.mdx (hunk 1 above)
Comment (pizzasaurusrex) [reply src/pages/docs/ssh/acls.mdx]: To be fair. I'm not either, I've just heard that phrase referenced a lot when customers asked for the feature

Comment (pizzasaurusrex) [reply src/pages/docs/ssh/acls.mdx]: going to just remove it.

**Review by tashian — APPROVED** [review]
lgtm

**Pull request merged**